azure/datalake/store/lib.py

```diff
@@ -25,6 +25,14 @@
 import requests
 import requests.exceptions
 
+# this is required due to github issue, to ensure we don't lose perf from openPySSL: https://github.com/pyca/pyopenssl/issues/625
+try:
+    from requests.packages.urllib3.contrib.pyopenssl import extract_from_urllib3 as enforce_no_py_open_ssl
+    enforce_no_py_open_ssl()
```

Comment:
I would move this into an `else` clause in case it triggers an `ImportError` which is an actual error and not something to ignore.